data.sql

```diff
@@ -9,4 +9,52 @@ INSERT INTO animals VALUES (7,'Squirtle','1993/04/02',3,FALSE,-12.13);
 INSERT INTO animals VALUES (8,'Angemon','2005/06/12',1,TRUE,-45);
 INSERT INTO animals VALUES (9,'Boarmon','2005/06/07',7,TRUE,20.4);
 INSERT INTO animals VALUES (10,'Blossom','1998/10/13',3,TRUE,17);
-INSERT INTO animals VALUES (11,'Ditto','2022/05/14',4,TRUE,22);
\ No newline at end of file
+INSERT INTO animals VALUES (11,'Ditto','2022/05/14',4,TRUE,22);
+
+-- Insert the following data into the `owners` table:
```

Comment:
- [ ] It would be nice if you make your PR Title such as `Vet clinic database: query multiple tables` instead of `Day 3` to make your PR Title descriptive 👍
- [ ] It would be nice if you make your branch name such as `feature/query_multiple_tables` instead of `day_3` to make your pull request more professional and descriptive 😊